### PR TITLE
enable anistropic window shape

### PR DESCRIPTION
**Pull request by grlee77**
I was unable to push directly to your branch, but if you merge this, the changes should show up in your scikit-image PR.

### CODE REVIEW

File: skimage/filters/_window.py [hunk 1]
```diff
@@ -82,37 +80,48 @@ def window(window_type, size, ndim=2, warp_kwargs=None):
...
 
     Return a Kaiser window with beta parameter of 16 and shape (256, 256, 256):
 
-    >>> w = window(16, 256, ndim=3)
+    >>> w = window(16, (256, 256, 256))
 
     Return a Tukey window with an alpha parameter of 0.8 and shape (100, 100):
 
-    >>> w = window(('tukey', 0.8), 100)
+    >>> w = window(('tukey', 0.8), (100, 100))
```
Comment (jni) [inline skimage/filters/_window.py:91]: Might as well change one of these examples to be anisotropic?

**Comment by jni** [comment]
Well, given @mellertd's results in scikit-image/scikit-image#4252, and the consistency of this API with other nD APIs, I kinda like this. @mellertd what do you think?

File: skimage/filters/_window.py (hunk 1 above)
Comment (mellertd) [reply skimage/filters/_window.py:91]: I have to merge in from master as I am hitting some build error on the travis CI, and will make this change at that time.

**Comment by mellertd** [comment]
I think this looks great -- I can still specify isotropic windows for my purposes and this lets people make anisotropic windows if they want them, and since the API is more consistent this is a win-win all around. Thanks again, @grlee77 !

File: skimage/filters/_window.py (hunk 1 above)
Comment (mellertd) [reply skimage/filters/_window.py:91]: done!

**Pull request merged**